assets/src/vo/index.js

```diff
@@ -4,4 +4,12 @@ export {
 	Currency,
 } from './currency';
 export { default as Label } from './label';
-export { DateTime, Duration, ServerDateTime } from './date-time';
+export {
+	DateTime,
+	Duration,
+	ServerDateTime,
+	DEFAULT_TIMEZONE_STRING,
+	DEFAULT_OFFSET,
+	DEFAULT_VALID_LOCALE,
+	DEFAULT_FORMAT,
```

Comment:
I just noticed late yesterday while troubleshooting another issue, that some similar constants are exported directly as part of the `eejs` object so I might be able to use those, although will need to investigate further to see if they offer the same datums.

Reply:
Ya there are some root constants that are available, but I think you may still want these exports because you'll likely reconstruct the same values elsewhere with the root constants.